rust/agents/scraper/src/db/block.rs

```diff
@@ -1,4 +1,4 @@
-//! SeaORM Entity. Generated by sea-orm-codegen 0.10.0
+//! SeaORM Entity. Generated by sea-orm-codegen 0.9.3
```

Comment:
Because 0.10.0, 0.10.1, and 0.10.2 generate code that does not compile. Also this upgrade that got reverted here was **not** intentional. What happened was I had forgotten to force the version in the script.